CHANGELOG.md

```diff
@@ -4305,7 +4305,7 @@
 * Added `String#insert`
 * Added `Time::EpochConverter` and `Time::EpochMillisConverter`
 * Added `%s` (unix epoch) directive to `Time::Format`
-* `Time` now honours Dayling Saving and `ENV["TZ"]`
+* `Time` now honours Daylight Saving and `ENV["TZ"]`
```

Comment:
Notable change.

Some projects don't like retroactively changing changelogs, if this is one, just let me know.

Reply:
I prefer fixing such typos in the changelog. Thanks!